site/content/contribute/mobile/e2e/ios.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Setup and Run iOS E2E"
+date: 2020-09-01T09:00:00-00:00
+weight: 4
+subsection: Mobile End-to-End Testing
+---
+
+#### Local setup
+
+1. Install [applesimutils](https://github.com/wix/AppleSimulatorUtils):
+```
+brew tap wix/brew
+brew install applesimutils
```

Comment:
is this required? can't we use the Xcode simulator commands for this? or does detox use this under the hood ?

Reply:
Detox is using this to control iOS simulator - start/stop/reload/install/uninstall/set permission etc.